bugbug/models/assignee.py

```diff
@@ -100,9 +104,9 @@ def get_labels(self):
             if count > MINIMUM_ASSIGNMENTS
         )
 
-        print(f"{len(top_assignees)} assignees")
+        logger.info("%s assignees", {len(top_assignees)})
         for assignee, count in assignee_counts:
-            print(f"{assignee}: {count}")
+            logger.info("%s: %s", assignee, count)
```

Comment:
```suggestion
            logger.info("%s: %d", assignee, count)
```

When the value is an integer, please use `%d`.

Reply:
Please apply the same change where applicable.

Reply:
I realized that a little later, thank you for the correction!